projects/pt1/e2e_testing/main.py

```diff
@@ -4,6 +4,7 @@
 # Also available under a BSD-style license. See LICENSE.
 
 import argparse
+import logging
```

Comment:
Some other projects to take inspiration from:

* [`shark_turbine/support/logging.py`](https://github.com/nod-ai/SHARK-Turbine/blob/main/core/shark_turbine/support/logging.py) and [`shark_turbine/support/debugging.py`](https://github.com/nod-ai/SHARK-Turbine/blob/main/core/shark_turbine/support/debugging.py)
* https://pytorch.org/docs/stable/logging.html

torch-mlir could share logging configuration code throughout the project or it could be isolated to just these testing folders. One advantage to doing it at the project level is that running both inside and outside of tests could get the same information. Could start with something in just the tests and see how it feels then generalize to the full project.